<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CSCValidator</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>16</y>
     <width>781</width>
     <height>601</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1366</width>
     <height>768</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>33</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_validate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>761</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Validate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_server">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>localhost:1666</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Branch</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>761</width>
      <height>331</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>759</width>
       <height>329</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="te_message">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="le_branch">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>//depot/</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_user">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>perforce</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_password">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>perforce</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>33</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_rule_file">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>customer.xml</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>csc_feature.xml</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>csc_feature_network.xml</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>default_application_order.xml</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>default_workspace.xml</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>apps.xml</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>others.xml</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File to check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_go">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sale code</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_sale">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_wsp">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>perforce_DESKTOP-6AAVS6M_7212</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Workspace</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_prj_type">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>RA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OS Upgrade</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Project type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Device type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_dev_type">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Cellular (4G)</string>
    </property>
    <item>
     <property name="text">
      <string>Cellular (4G)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WIFI</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SIM type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_sim_type">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Single SIM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dual SIM</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="gb_options">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>99</y>
      <width>761</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color: rgb(178, 225, 255);
	font-size: 11px;
    font-weight: bold;
}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px;
    color:blue
}</string>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QCheckBox" name="cb_ims_supported">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>IMS supported</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Version: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_version_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>550</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lam.bich</string>
    </property>
   </widget>
   <zorder>gb_options</zorder>
   <zorder>label</zorder>
   <zorder>pb_validate</zorder>
   <zorder>le_server</zorder>
   <zorder>label_3</zorder>
   <zorder>scrollArea</zorder>
   <zorder>le_branch</zorder>
   <zorder>le_user</zorder>
   <zorder>le_password</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>cb_rule_file</zorder>
   <zorder>label_8</zorder>
   <zorder>pb_go</zorder>
   <zorder>label_9</zorder>
   <zorder>cb_sale</zorder>
   <zorder>le_wsp</zorder>
   <zorder>label_4</zorder>
   <zorder>cb_prj_type</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>cb_dev_type</zorder>
   <zorder>label_12</zorder>
   <zorder>cb_sim_type</zorder>
   <zorder>label_version</zorder>
   <zorder>label_version_2</zorder>
  </widget>
  <widget class="QGroupBox" name="gb_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>761</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    background-color:rgb(255, 157, 218);
	font-size: 11px;
    font-weight: bold;
}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px;
    color:blue
}</string>
   </property>
   <property name="title">
    <string>Connection</string>
   </property>
  </widget>
  <zorder>gb_input</zorder>
  <zorder>centralwidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
